### PR TITLE
fix: remove unread hono vars

### PR DESCRIPTION
**Pull request by danielgent**
I'll leave this until all the other PRs are merged

### CODE REVIEW

File: src/authentication-flows/silent.ts [hunk 1]
```diff
@@ -45,8 +45,6 @@ export async function silentAuth({
 }: SilentAuthParams) {
   const { env } = ctx;
 
-  ctx.set("logType", LogTypes.SUCCESS_SILENT_AUTH);
```
Comment (danielgent) [inline src/authentication-flows/silent.ts:48]: we're explicitly logging this now below

File: src/types/Var.ts [hunk 2]
```diff
@@ -2,12 +2,8 @@ import { LogType, Auth0Client } from "./auth0";
 import { z } from "zod";
 
 export type Var = {
-  vendorId?: string;
   startAt: number;
   userId?: string;
-  tenantId?: string;
-  log: string;
-  logType?: LogType;
```
Comment (danielgent) [inline src/types/Var.ts:10]: we're not reading `vendorId` or `tenantId`, but we could still keep them (or read them... but doesn't seem worth it)

but `log` and `logType` are not being used now :+1:

File: src/routes/management-api/users.ts [hunk 3]
```diff
@@ -293,7 +293,6 @@ export const userRoutes = new OpenAPIHono<{ Bindings: Env; Variables: Var }>()
       });
 
       ctx.set("userId", data.id);
-      ctx.set("tenantId", tenant_id);
```
Comment (danielgent) [inline src/routes/management-api/users.ts:296]: We specify this inline with `ctx.env.data.logs.create(tenant_id,`

File: src/middlewares/authentication.ts [hunk 4]
```diff
@@ -164,7 +164,6 @@ export default function authenticationMiddleware(
           azp: token.payload.azp || "sesamy",
           permissions,
         });
-      ctx.set("vendorId", token.payload.azp || "sesamy");
```
Comment (danielgent) [inline src/middlewares/authentication.ts:167]: this isn't read anywhere. this var is now removed from the type

**Review by markusahlstrand — APPROVED** [review]
Looks good.

**Pull request merged**